### PR TITLE
[AN-4639] Adds visual tweaks for ephemeral mode

### PR DESCRIPTION
**Pull request by shumengye**
#### Description
In ephemeral mode, all cursor icons are now in accent color as well as placeholders for expired ephemeral messages.

Removes yellow accent color from picker in settings.

#### Ticket
https://wearezeta.atlassian.net/browse/AN-4639
#### APK
[Download build #7808](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7808/artifact/build/artifact/wire-dev-PR278-7808.apk)
[Download build #7809](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7809/artifact/build/artifact/wire-dev-PR278-7809.apk)
[Download build #7811](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7811/artifact/build/artifact/wire-dev-PR278-7811.apk)
[Download build #7812](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7812/artifact/build/artifact/wire-dev-PR278-7812.apk)
[Download build #7817](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7817/artifact/build/artifact/wire-dev-PR278-7817.apk)
[Download build #7821](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7821/artifact/build/artifact/wire-dev-PR278-7821.apk)
[Download build #7827](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7827/artifact/build/artifact/wire-dev-PR278-7827.apk)
[Download build #7828](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7828/artifact/build/artifact/wire-dev-PR278-7828.apk)
[Download build #7831](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7831/artifact/build/artifact/wire-dev-PR278-7831.apk)
[Download build #7832](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7832/artifact/build/artifact/wire-dev-PR278-7832.apk)

### CODE REVIEW

**Comment by shumengye** [comment]
build again

**Comment by shumengye** [comment]
build again

**Comment by xuanzhaopeng** [comment]
Emoji button besides cursor input doesn't work when I change to Ephemeral mode. (Confirmed it works in current exp build 1306)

File: wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java [hunk 1]
```diff
@@ -244,7 +252,8 @@ public void onClick(View view) {
                     return;
                 }
                 mainToolbar.unselectItems();
-                if (getContext().getString(R.string.glyph__emoji).equals(emojiButton.getText())) {
+                if (getContext().getString(R.string.glyph__emoji).equals(emojiButton.getText()) ||
```
Comment (shumengye) [inline wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java]: good point, will do

**Pull request merged**